src/parsers/index.ts

```diff
@@ -13,6 +13,5 @@ export {
   defaultGasParamsFormat,
   parseRainbowMeteorologyData,
 } from './gas';
-export { parseNewTransaction } from './newTransaction';
 export { getRequestDisplayDetails } from './requests';
-export { getDescription } from './transactions';
+export { getDescription, convertNewTransactionToRainbowTransaction } from './transactions';
```

Comment:
renamed `parseNewTransaction` to `convertNewTransactionToRainbowTransaction` to properly describe what it does